### PR TITLE
Pull request for WAZO-1179-tenant-id-to-uuid

### PR DESCRIPTION
**Pull request by sileht**
## tests: fix database migration creation

Since database creation have been done by session, the database
migration was tested against an already created database.

This change fix it to run it on a fresh one

## mypy: ignore some issues

mypy doesn't detect the type when it's on the next line...

### CODE REVIEW

**Comment by sileht** [comment]
recheck

**Review by mwolff44 — DISMISSED** [review]

**Comment by sileht** [comment]
recheck

**Comment by sileht** [comment]
recheck

**Comment by sileht** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

**Comment by sileht** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

File: wazo_router_confd/database.py [hunk 1]
```diff
@@ -74,11 +74,12 @@ def __init__(self, dsn):
 
 def from_database_uri_to_dsn(database_uri: str) -> str:
     parsed_uri = urlparse(database_uri)
-    dsn = 'dbname=%s user=%s password=%s host=%s' % (
+    dsn = 'dbname=%s user=%s password=%s host=%s port=%d' % (
```
Comment (tranchitella) [inline wazo_router_confd/database.py:77]: Good catch! I missed it previously.

**Review by tranchitella — APPROVED** [review]
Everything looks good!

**Pull request merged**